simplecasper/app.py

```diff
@@ -126,5 +148,230 @@ def run(ctx, node_id, console):
     app.stop()
 
 
```

Comment:
Code below this point from https://github.com/ethereum/pyethapp/blob/develop/pyethapp/app.py#L458